cmd/cli/app/repo/repo_register.go

```diff
@@ -171,10 +197,9 @@ func getSelectedRepositories(repoList []*minderv1.UpstreamRepositoryRef, flagRep
 	// Create a slice of strings to hold the selected repos
 	var allSelectedRepos []string
 
-	// If the --repo flag is set, use it to select repos
-	if flagRepos != "" {
-		repos := strings.Split(flagRepos, ",")
-		for _, repo := range repos {
+	// If the --name flag is set, use it to select repos
+	if len(inputRepositories) != 0 {
```

Comment:
Again, we probably don't need `len()` here, just use the loop iteration.

Reply:
Right 🤦🏽‍♂️ . Done.